ethcore/src/client/chain_notify.rs

```diff
@@ -114,19 +114,51 @@ impl ChainRoute {
 	}
 }
 
+/// Used by `ChainNotify` `new_blocks()`
+pub struct NewBlocks {
+	/// Imported blocks
+	pub imported: Vec<H256>,
+	/// Invalid blocks
+	pub invalid: Vec<H256>,
+	/// Route
+	pub route: ChainRoute,
+	/// Sealed
+	pub sealed: Vec<H256>,
+	/// Block bytes.
+	pub proposed: Vec<Bytes>,
+	/// Duration
+	pub duration: Duration,
+	/// `Validation.processing` is empty?
+	pub processing_is_empty: bool,
```

Comment:
I'm not fully sold on the name, I'd prefer something less technical and more descriptive, like maybe: `has_more_blocks_to_import`?

Reply:
actually that's a good call, because it also reverses the polarity of the bool which makes it more intuitive to use